### DISCUSSION TITLE
"Tentative" vote type

### DISCUSSION THREAD

**Discussion by qhantom** [lukevella/rallly#216] [Ideas]
Hey there!

I had the idea to implement a vote type "tentative". I already finished it but I'm not sure whether it's a feature for everyone. Is it welcome to open a pull request for that option?

qhantom

**Comment by lukevella** [comment]
Hi @qhantom, has already been implemented. See #147

**Reply by qhantom** [reply]
In my definition "tentative" has a different meaning than "if need be". In my company sometimes it's useful to vote with an option that expresses "I'm not quite sure yet". Maybe that's too specific.

**Reply by lukevella** [reply]
Maybe in the future I'll make it possible to change the definition of the tentative vote from "if need be" to "maybe" but it's not a priority. You can choose to interpret the votes as you wish.

**Reply by qhantom** [reply]
Alright, thanks! =)
